hedera-mirror-rest/api/v1/openapi.yml

```
@@ -1656,6 +1686,92 @@ components:
       items:
         type: string
       type: array
+    ContractAction:
+      type: object
+      properties:
+        call_depth:
+          description: The nesting depth of the call
+          example: 1
+          format: int32
+          nullable: false
+          type: integer
+        call_operation_type:
+          description:  The type of the call operation
+          enum: [OP_UNKNOWN, OP_CALL, OP_CALLCODE, OP_DELEGATECALL, OP_STATICCALL, OP_CREATE, OP_CREATE2]
+          example: OP_CALL
+          nullable: false
+          type: string
+        call_type:
+          description: The type of the call
+          enum: [NO_ACTION, CALL, CREATE, PRECOMPILE, SYSTEM]
+          example: CALL
+          nullable: false
+          type: string
+        caller:
+          $ref: '#/components/schemas/EntityId'
+        caller_type:
+          description: The entity type of the caller
+          type: string
+          enum: [ ACCOUNT, CONTRACT ]
+          example: ACCOUNT
+          nullable: false
+        from:
+          description: The EVM address of the caller
+          example: "0x0000000000000000000000000000000000000065"
+          nullable: false
+          type: string
+        gas:
+          description: Gas cost in tinybars
+          example: 50000
+          nullable: false
+          format: int64
+          type: number
```

See https://swagger.io/docs/specification/data-models/data-types/#numbers
```suggestion
          type: integer
```